scenes/game/local_game.gd

```diff
@@ -3543,8 +3568,14 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 				decision_info.player = performing_player.my_id
 				decision_info.valid_zones = valid_zones
 				decision_info.limitation = effect['limitation']
-				performing_player.sustain_next_boost = true
+				decision_info.ignore_costs = 'ignore_costs' in effect and effect['ignore_costs']
+				var sustain = true
+				if 'sustain' in effect and not effect['sustain']:
+					sustain = false
+				performing_player.sustain_next_boost = sustain
 				performing_player.cancel_blocked_this_turn = true
+				# Mid-strike boosts cannot do extra actions or strikes or reading.
+				performing_player.no_extra_actions_or_strike_this_turn = true
```

Comment:
Should maybe also apply this to the additional_boost effect if active_strike is true (is there a particular reason for adding this check instead of checking active_strike directly?)